src/app/components/footer/footer.component.html

```diff
@@ -0,0 +1,20 @@
+<section class="footer-section">
+  <div class="left-section">
+    <div class="elements-container">
+      <span class="logo-wolox"></span>
+    </div>
+    <div class="elements-container">
+      <div class="social-media">
+        <span class="linked-in social-icon"></span>
+        <span class="social-text">Conecta con nostros</span>
+      </div>
+    </div>
```

Comment:
Check if you really need the `elements-container` wrapper 

```suggestion
      <span class="logo-wolox"></span>
      <div class="social-media">
        <span class="linked-in social-icon"></span>
        <span class="social-text">Conecta con nostros</span>
      </div>
```